internal/xds/translator/oidc.go

```diff
@@ -180,6 +180,13 @@ func oauth2Config(oidc *ir.OIDC) (*oauth2v3.OAuth2, error) {
 			Resources:  oidc.Resources,
 
 			PreserveAuthorizationHeader: preserveAuthorizationHeader,
+
+			RetryPolicy: &corev3.RetryPolicy{
```

Comment:
should be opt in, imo, to maintain the same API behavior with xRoutes

Reply:
Discussed this with @arkodg this morning, it's better to put RetryPolicy into the `ClusterSettings` and use `BackendRef` to model the Auth server. In this way, the RetryPolicy can also be reused by jwt and other features.